cogs/polls.py

```diff
@@ -2,23 +2,29 @@
 from discord.ext import commands
 import random
 import json
+from typing import Final, Dict
 
-class Polls(commands.Cog):
-    
-    def __init__(self, client):    
-        self.client = client
+class PollsCog(commands.Cog):
+    POLLS_CHANNEL_ID : Final[int] = 962433889081626624
```

Comment:
Channel should preferably not be hard coded. Also this is a configuration that should be set in the config.yaml. You want to specify a channel name.

Another option would be to have a set_poll_channel() function, which takes a channel as argument. This is than saved to then data folder. This should be individual for every server. I think this would be the nicer solution.

Do this in a follow up change, and reply to this comment with that pull request. Only when replied you can close this comment.

You can also create a ticket for it and close this comment. In that case link the ticket. Your choice.

Reply:
For now, I will set the POLLS_CHANNEL_ID in the config. We could do a follow up change to support multiple polls for each server